scripts/prepare_alpaca.py

```diff
@@ -54,9 +54,9 @@ def prepare(
     tokenizer = Tokenizer(checkpoint_dir)
 
     # Partition the dataset into train and test
-    train_split_size = len(data) - test_split_size
+    train_split_percent = 100 - test_split_percent
```

Comment:
```suggestion
    train_split_percent = 1.0 - test_split_percent
```

Reply:
Yes, changing it to a fraction makes sense, but the variable names should also be changed, as they are no longer percentages.  Percent comes from the Latin phrase "per centum" which means "by the hundred" or "for each hundred."